github-management/opening-a-request.md

```diff
@@ -19,7 +19,7 @@ issue. If your request is urgent, please escalate to **[@kubernetes/owners]**.
 If you need help with the following:
 - Bot configuration
 - Automatic merging
-- Label issues
+- Issue labelling
```

Comment:
FWIW there are more occurrences of the American English "labeling" than there are "labelling" in this repo, but I'm not going to object if this merges by someone else's /lgtm

Reply:
My general take on these types of language differences: it's the author's preference and either is correct. It's how I'm injecting a little bit of 🇨🇦 into these docs 😁